### PR TITLE
Always add a temporary Swift version to generated app target

### PR DESCRIPTION
**Pull request by dnkoutso**
Fixes a case where a podspec does not specify a Swift version in its DSL and instead uses `.swift-version`.

### CODE REVIEW

File: lib/cocoapods/generate/installer.rb [hunk 1]
```diff
@@ -109,7 +109,11 @@ def create_app_project
         spec.available_platforms.map do |platform|
           consumer = spec.consumer(platform)
           target_name = "App-#{Platform.string_name(consumer.platform_name)}"
-          Pod::Generator::AppTargetHelper.add_app_target(app_project, consumer.platform_name, deployment_target(consumer), target_name)
+          native_app_target = Pod::Generator::AppTargetHelper.add_app_target(app_project, consumer.platform_name, deployment_target(consumer), target_name)
+          # Temporarily set Swift version to pass validator checks for pods which do not specify Swift version.
+          # It will then be re-set again within #perform_post_install_steps.
+          Pod::Generator::AppTargetHelper.add_swift_version(native_app_target, '3.2')
```
Comment (segiddins) [inline lib/cocoapods/generate/installer.rb]: please extract this out into a constant

Comment (dnkoutso) [reply lib/cocoapods/generate/installer.rb]: should I reuse the same one from `validator.rb` ?

Comment (dnkoutso) [reply lib/cocoapods/generate/installer.rb]: https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/validator.rb#L19

Comment (segiddins) [reply lib/cocoapods/generate/installer.rb]: 👍

Comment (dnkoutso) [reply lib/cocoapods/generate/installer.rb]: done

**Pull request merged**